PlaceRouteHierFlow/PnRDB/datatype.h

```diff
@@ -375,6 +377,7 @@ struct hierNode {
   string gdsFile="";
   vector<int> parent;
   vector<blockComplex> Blocks;
+  map<string, int> Block_name_map;//map from block name to block index
```

Comment:
Doesn't look like you actually need this? You set it but don't read from it. Is that correct? You may still want it for something later.